test/metabase/api/public_test.clj

```diff
@@ -1757,68 +1757,70 @@
-                                        (format "public/action/%s/execute" public_uuid)
-                                        {:parameters {:id 1 :name "European"}})]
-                (is (str/starts-with? (:body throttled-response) "Too many attempts!"))
-                (is (contains? (:headers throttled-response) "Retry-After"))))))
-        ;; Lift the throttle attempts threshold so we don't have to wait between requests
-        (with-redefs [api.public/action-execution-throttle (throttle/make-throttler :action-uuid :attempts-threshold 1000)]
-          (mt/with-actions [{} (assoc action-opts :archived true)]
-            (testing "Check that we get a 400 if the action is archived"
-              (is (= "Not found."
-                     (client/client
-                      :post 404
-                      (format "public/action/%s/execute" (str (random-uuid)))
-                      {:parameters {:id 1 :name "European"}})))))
-          (mt/with-actions [{} action-opts]
-            (testing "Check that we get a 400 if the action doesn't exist"
-              (is (= "Not found."
-                     (client/client
-                      :post 404
-                      (format "public/action/%s/execute" (str (random-uuid)))
-                      {:parameters {:id 1 :name "European"}}))))
-            (testing "Check that we get a 400 if sharing is disabled."
-              (mt/with-temporary-setting-values [enable-public-sharing false]
-                (is (= "An error occurred."
-                       (client/client
-                        :post 400
-                        (format "public/action/%s/execute" public_uuid)
-                        {:parameters {:id 1 :name "European"}})))))
-            (testing "Check that we get a 400 if actions are disabled for the database."
-              (mt/with-temp-vals-in-db Database (mt/id) {:settings {:database-enable-actions false}}
-                (is (= "An error occurred."
-                       (client/client
-                        :post 400
-                        (format "public/action/%s/execute" public_uuid)
-                        {:parameters {:id 1 :name "European"}})))))
-            (testing "Check that we send a snowplow event when execute an action"
-              (snowplow-test/with-fake-snowplow-collector
-                (client/client
-                 :post 200
-                 (format "public/action/%s/execute" public_uuid)
-                 {:parameters {:id 1 :name "European"}})
-                (is (= {:data   {"action_id" (t2/select-one-pk 'Action :public_uuid public_uuid)
-                                 "event"     "action_executed"
-                                 "source"    "public_form"
-                                 "type"      "query"}
-                        :user-id nil}
-                       (last (snowplow-test/pop-event-data-and-user-id!))))))))))))
+  (mt/with-premium-features #{:advanced-permissions}
```

Comment:
This is a short-term fix for this test. We will investigate a way to set up CI better as a follow-up to avoid this class of issues